frontend/.env.schema.json

```
@@ -30,6 +30,14 @@
       "required": true,
       "secret": false,
       "example": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9..."
+    },
+    {
+      "name": "VITE_GENERATION_ENABLED",
+      "description": "Toggle video generation. Set to false to suspend generation while LangGraph Cloud is stopped (cost saving). Defaults to disabled when unset.",
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The "description": "Toggle video generation. Set to false to suspend generation while LangGraph Cloud is stopped (cost saving). Defaults to disabled when unset.", key has incorrect delimiter

(IncorrectDelimiter)